src/components/app/MenuButtons/MetaInfo.js

```diff
@@ -87,6 +87,32 @@ export class MenuButtonsMetaInfo extends React.PureComponent<Props> {
 
     const platformInformation = formatPlatform(meta);
 
+    let cpuCount = null;
+    let physicalCPUs = null;
+    let logicalCPUs = null;
+    if (meta.physicalCPUs || meta.logicalCPUs) {
+      if (meta.physicalCPUs) {
+        physicalCPUs =
+          meta.physicalCPUs +
+          ' physical ' +
+          (meta.physicalCPUs === 1 ? 'core' : 'cores');
+      }
+      if (meta.logicalCPUs) {
+        logicalCPUs =
+          meta.logicalCPUs +
+          ' logical ' +
+          (meta.logicalCPUs === 1 ? 'core' : 'cores');
+      }
+      cpuCount = (
+        <div className="metaInfoRow">
+          <span className="metaInfoLabel">CPU:</span>
+          {physicalCPUs ? physicalCPUs : null}
```

Comment:
You don't need to do this check as well, you can directly print `physicalCPUs` instead.